packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1107,6 +1107,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
         calloutWidth={useComboBoxAsMenuWidth && this._comboBoxWrapper.current ? comboBoxMenuWidth && comboBoxMenuWidth : dropdownWidth}
         calloutMaxWidth={dropdownMaxWidth ? dropdownMaxWidth : comboBoxMenuWidth}
         hidden={persistMenu ? !isOpen : undefined}
+        popupShouldNotRestoreFocus={this.props.menuShouldNotRestoreFocus}
```

Comment:
Same thing here, I think we need to be consistent with naming conventions.

Reply:
As you pointed out above, I was trying to maintain the current behavior where undefined = true.  I'll go with keeping the names the same, though.